compiler/ras/DebugCounter.cpp

```diff
@@ -462,6 +463,33 @@ void TR::DebugCounterAggregation::accumulate()
       }
    }
 
+
+int64_t TR::DebugCounterAggregation::getCount()
+   {
+   int64_t count = 0;
+   ListIterator<CounterDelta> it(_counterDeltas);
+
+   for (CounterDelta *counterDelta = it.getFirst(); counterDelta; counterDelta = it.getNext())
+      {
+      count += counterDelta->counter->getCount();
+      }
+
+   return count;
+   }
+
+void TR::DebugCounterAggregation::printCounters()
+   {
+   ListIterator<CounterDelta> it(_counterDeltas);
+
+   for (CounterDelta *counterDelta = it.getFirst(); counterDelta; counterDelta = it.getNext())
+      {
+      TR::DebugCounter *counter = counterDelta->counter;
+      int64_t count = counter->getCount();
+      if (count)
+         TR_VerboseLog::writeLineLocked(TR_Vlog_PERF, "RSS counter count=%d %s", count, counter->getName());
```

Comment:
"RSS counter" : can a more generic message be printed such that these functions are not tied to the RSS report?

Reply:
Fixed.